Sources/JavaKit/generated/JavaOptional.swift

```diff
@@ -0,0 +1,51 @@
+// Auto-generated by Java-to-Swift wrapper generator.
+import JavaRuntime
+
+@JavaClass("java.util.Optional")
+open class JavaOptional<T: AnyJavaObject>: JavaObject {
+  @JavaMethod
+  open func get() -> JavaObject!
+
+  @JavaMethod
+  open override func equals(_ arg0: JavaObject?) -> Bool
+
+  @JavaMethod
+  open override func toString() -> String
+
+  @JavaMethod
+  open override func hashCode() -> Int32
+
+  @JavaMethod
+  open func isEmpty() -> Bool
+
+  @JavaMethod
+  open func isPresent() -> Bool
+
+  @JavaMethod
+  open func orElse(_ arg0: JavaObject?) -> JavaObject!
+
+  @JavaMethod
+  open func orElseThrow() -> JavaObject!
+}
+extension JavaClass {
```

Comment:
Does this look right? Shouldn't this be JavaClass<JavaOptional> ?

Reply:
Hmmm, I see (note: I've not spend much time with JavaKit but more focused on the opposite direction, so thanks for helping review this 😄)... 

Yeah that makes sense I guess, I wonder if this will become a problem in the long term with names clashing -- lots of types like to have a `of(_:)` or some form of `create(_:)` but maybe it'll work out okey, could be a lot of overloads eventually hm